<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>401</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>3ds</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>VK</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/pics/pic.ico</normaloff>:/pics/pic.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="startShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始显示</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoStart">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>开机自启动</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>VisualKeyboard</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示通知</string>
    </property>
   </widget>
   <widget class="QLabel" name="bgColorTip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>背景颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="wdColorTip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>字体颜色</string>
    </property>
   </widget>
   <widget class="QtClickableLabel" name="bgColor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QtClickableLabel" name="wdColor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QtClickableLabel" name="CurrentPos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>开始位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdity">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultPos">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>默认</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="author"/>
    <addaction name="donate"/>
    <addaction name="update"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="author">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="donate">
   <property name="text">
    <string>捐赠</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>检查更新</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtClickableLabel</class>
   <extends>QLabel</extends>
   <header>qtclickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
    <slot>mousePressEvent(QMouseEvent*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
